docs/workforce-management-framework/topics/con-workflow-step.adoc

```diff
@@ -0,0 +1,60 @@
+= Workflow Step
+
+== Step Structure
+Steps are divided into to linked catergories the `module` and the `definition`.
```

Comment:
Can you include a 1 liner for "linked categories" - as a reader, I don't know that this is.

Reply:
Already covered bellow.

Reply:
There is no other mention of "linked catergories", "linked" or "catergories" anywhere else in this document.

According to the docs, "Edit Section (Form)" and "Preview Section (View)" are "two types of directive".

Are these the "linked catergories"?

Reply:
Right. I thought ""linked categories" was some sort of technical definition. So on that basis, any of the following reworks of the original sentence appeal to you:

A) "A _Step_ is divided into two categories, the `module`  category and the `definition` category and both categories are linked."

B) "A _Step_ consists of a `module` and a `definition`."

C) "A _Step_ consists of a `module` and a `definition`, and both are linked."

Reply:
The module and the definition can be expanded to have more that one step. so using the singular step would be incorrect.